src/App.js

```diff
@@ -5,7 +5,7 @@ import WrappedNormalLoginForm from './component/Login.js';
 import error from './component/error.js';
 import WrappedRegisterForm from './component/Register.js';
```

Comment:
no need to write .js
eg - import WrappedAboutForm from './component/About
this will also work

Reply:
ok sir